screengrab/lib/screengrab/page.html.erb

```diff
@@ -64,117 +64,99 @@
       }
     </style>
   </head>
-  <body>
-    <% divide_size_by = 5.0 %>
-    <% max_width = 180 %>
-    <% image_counter = 0 %>
+  <body><% image_counter = 0 %><% @data.each do |language, content| %>
```

Comment:
Had to put all of those `<% .. %>` on the same line in order to not generate new lines. Is there any other way around this?